eslint-example/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+    "manifest_version": 2,
+    "description": "Example using eslint",
+    "name": "eslint-example",
+    "version": "1.0",
+    "homepage_url": "https://github.com/mdn/webextensions-examples/tree/master/eslint-example",
+
+    "applications": {
```

Comment:
This applications section can be removed. It's not needed for this example so removing it would be more helpful to someone copying/pasting this code. When you omit this section, the extension can be compatible with Chrome.